h2o-core/src/main/java/water/util/Log.java

```diff
@@ -22,261 +23,234 @@
  *  logging, no guarantees are made about the messages.
  **/
 abstract public class Log {
+  private static final byte UNKNOWN = -1;
```

Comment:
Yes, that was my first approach. However I wasn't sure about the performance. I wanted to replace numeric log levels by enums. Do you think it's fine to do ? I would like that better as well.